shells/ngs/Portfile

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake       1.1
+PortGroup           github      1.0
+
+github.setup        ngs-lang ngs 0.2.13 v
+github.tarball_from archive
+revision            0
+
+homepage            https://ngs-lang.org
+
+description         \
+    Next Generation Shell (NGS)
+
+long_description    \
+    NGS is an alternative shell. At its core is a domain-specific language \
+    that was specifically designed to be a shell language. It is designed \
+    with a focus on Ops and systems engineering. One way to think about NGS \
+    is bash plus data structures plus better syntax and error handling.
+
+checksums           rmd160  ea7335942152c9ae74d3f240077e4063bc592dbe \
+                    sha256  7648761edb3695292d3289b91f9644c204d42269b8af697c765707ce192e45b5 \
+                    size    724109
+
+categories          shells
+installs_libs       no
+license             GPL-3
+maintainers         {gmail.com:herby.gillot @herbygillot} \
+                    openmaintainer
+
+patchfiles          patch-implicit.diff
+
+post-patch {
+    # Replace hard-coded prefixes
+    foreach ftarget {
+        doc/ngstut.1.md
+        doc/ngs.1.md
+        doc/ngslang.1.md
+        lib/autoload/globals/DelimStr.ngs
+        lib/stdlib.ngs
+        CMakeLists.txt
+    } {
+        reinplace -E "s|/usr/local|${prefix}|g" ${worksrcpath}/${ftarget}
+    }
+
+    # Correct man and doc paths
+    reinplace {s|DESTINATION man/man1|DESTINATION share/man/man1|} \
+        ${worksrcpath}/CMakeLists.txt
+
+    reinplace {s|DESTINATION doc/ngs|DESTINATION share/doc/ngs|} \
+        ${worksrcpath}/CMakeLists.txt
+}
+
+depends_build-append \
+                    path:bin/gsed:gsed  \
+                    port:pandoc         \
+                    port:peg
+
+depends_lib-append  port:boehmgc    \
+                    port:json-c     \
+                    port:libffi     \
+                    port:pcre2
+
+configure.env-append \
+                    "PKG_CONFIG_PATH=${prefix}/lib/pkgconfig"
+
+build.env-append    "PKG_CONFIG_PATH=${prefix}/lib/pkgconfig"
```

Comment:
this seems unused since you don't have a dependency on `pkgconfig`?

Reply:
🤦🏾‍♂️ Good catch @reneeotten. Added.